src/DGtal/io/readers/ITKReader.h

```diff
@@ -104,7 +105,7 @@ namespace DGtal
               typename ITKIOTrait<typename TImage::Value>::DefaultReadFunctor>
     static Image importITK(
     const std::string & filename,
-    const TFunctor & aFunctor = TFunctor() );
+    const TFunctor & aFunctor = TFunctor(), bool shiftDomain=true);
```

Comment:
What do you think about changing the name to `shiftDomainUsingOrigin`
And what about setting the default to false?